.php_cs.dist

```diff
@@ -39,21 +46,21 @@ $rules = [
 //    'php_unit_strict' => true,
 ];
 
-if (\PHP_VERSION_ID >= 50600) {
+if (version_compare($targetPhpVersion, '5.6', '>=')) {
```

Comment:
Imo it makes no sense to have conditionals on this file. A branch should have its own ruleset and it should be regardless of what version of php do you use to lint it

Reply:
So you would use twig to generate that file, right? I think this is a better solution indeed, what I did is dirty.

Reply:
@jordisala1991 I don't see the issue to have conditional statement. The configuration is the PHP file, we can take benefit from it.

BTW, moving it to a twig file on dev-kit is just make it dynamic else where.

Reply:
Yeah but see how I'm getting `$targetPhpVersion` : yuck! plus now it requires people only ever use `make` to run the cs-fixer